src/marqo/tensor_search/tensor_search.py

```diff
@@ -370,17 +373,23 @@ def add_documents(config: Config, add_docs_params: AddDocsParams):
                 continue
             chunk_values_for_filtering[key] = value
 
+        def is_non_tensor_field(field: str) -> bool:
```

Comment:
since this is repeated logic, it may be worth making it a util func that accepts: `field`, `tensor_fields (which may be None)`, and `non_tensor_fields (which may be None)`

Reply:
I like this idea!